config/locales/en.json

```
@@ -188,11 +188,12 @@
 	"conditions.save_condition": "Save condition",
 	"conditions.symptoms_began": "Symptoms began: ",
 	"conditions.title": "Medical conditions",
-	"confirmation.intro": "Confirmation Intro goes here",
-	"confirmation.save_data": "Save data",
-	"confirmation.thank_you": "Thank you!",
-	"confirmation.thank_you_participating": "Thank you for participating!",
-	"confirmation.title": "Confirmation Title",
+	"confirmation.title": "You have submitted %s's medical report",
+	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits",
```

```suggestion
	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits.",
```
Just adds a full stop/period to the end.